inc/class-s3-uploads-stream-wrapper.php

```diff
@@ -208,14 +210,19 @@ public function stream_flush()
 			}
 		}
 
+		// Post ID (e.g., for an attachment) on the request which originated the file, if applicable
+		$filename = end(explode('/', $params['Key']));
```

Comment:
Use the PHP built in basename function. Never know when windows will strike with '/'.